packages/desktop-client/src/components/reports/spreadsheets/summary-spreadsheet.ts

```
@@ -0,0 +1,213 @@
+
+      case 'avgPerTransact':
+        setData({
+          total:
+            (data.data[0].count
+              ? data.data[0].amount / data.data[0].count
+              : 0) / 100,
+        });
+        break;
+
+      case 'avgPerMonth':
+        const months = getOneDatePerMonth(startDay, endDay);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)